### DISCUSSION TITLE
Windows 11 unrecoverable after installing

### DISCUSSION THREAD

**Discussion by DominickVale** [valinet/ExplorerPatcher#2167] [Q&A]
I tried to install this patcher on my windows 11 setup to get my taskbar moved on the left. Turned out to be a mistake since the taskbar didn't work anymore, i had to uninstall it and now the normal windows 11 taskbar is working but the file explorer is bugged out like this:

![image](https://github.com/valinet/ExplorerPatcher/assets/52491108/699caca1-d33d-401d-b661-c87f9c8b9186)

Please help i really didn't need to waste my time like this just to get a stupid taskbar on the left. Fuck windows.

**Comment by DominickVale** [comment]


**Reply by Poopooracoocoo** [reply]
Surely by malware you're referring to Windows and not ExplorerPatcher.

**Reply by BraINstinct0** [reply]
Oh? If you think so why risk downloading a "malware" voluntarily and install it willingly. Fiddling with system components is minimum risky, and usually has risk of going haywire. Should have lived a smooth life if this is the most stuck thing in life...

**Reply by Poopooracoocoo** [reply]
ExplorerPatcher certainly isn't malware. It's a marvel in reverse engineering. It's amazing that Valentin brought the Windows 10 taskbar and Start Menu to Windows 11, and it's amazing that Amrsatrio made it work on OS Build 22621.2283. It's sad that we even need ExplorerPatcher to do what we could in Windows 10

By the way you never stated which build you were on and which version of EP you had. 

Personally I've never had EP cause a BSOD.

**Reply by Amrsatrio** [reply]
You could've waited for a friendly assistance from us instead of ultimately breaking your system like that... We could've told you to run `rundll32 C:\Windows\dxgi.dll,ZZGUI` and disabling this:

![image](https://github.com/valinet/ExplorerPatcher/assets/20662640/415e636e-697b-4746-9dfd-56d20da342c6)

I've been using EP since March 2022 and I've never had BSODs or such too. The worst I've had (and most people) is Explorer refusing to start up at all after an update until certain tricks are carried out, which I've fixed as my contribution to this project. 

But, you do you.

**Comment by DominickVale** [comment]


**Reply by BraINstinct0** [reply]
Well if you want proper assistance, I'd assume you'd at least included your system details, some more descriptions of your situation and steps to reproduce it. We can't simply magically "understand" your issue and what's going on your system without you explaining.

A good start is reading _other issues_ to get an idea of what information are usually asked, so you can add it to help quicken the troubleshooting.

**Reply by LQZ77** [reply]
这个软件和windows之间有一些bug，例如闪白色屏幕，黑屏，打不开管理器等等，我可以解决他，通过很多种bug方式去解决。
